app/scripts/navigation/navigation.controller.js

```diff
@@ -12,10 +12,10 @@
    * # MainCtrl
    * Controller of the Navbar on the top
    */
-  NavigationController.$inject = ['$location'];
+  NavigationController.$inject = ['$location', 'cartService', '$rootScope'];
```

Comment:
Instead of service injection please create a cartStatus directive and add it to the navigation template